source/server-discovery-and-monitoring/server-discovery-and-monitoring.md

```diff
@@ -485,7 +486,13 @@ removed once the primary is checked.
 #### error
 
 If the client experiences any error when checking a server, it stores error information in the ServerDescription's error
-field.
+field. The message contained in this field MUST contain the substrings detailed in the table below when the
+ServerDescription is changed to Unknown in the circumstances outlined.
+
+| circumstance                                               | error substring                                                |
+| ---------------------------------------------------------- | -------------------------------------------------------------- |
+| RSPrimary with a stale electionId/setVersion is discovered | `'primary marked stale due to electionId/setVersion mismatch'` |
```

Comment:
In this comment https://github.com/mongodb/specifications/pull/1729#discussion_r1917472576 I had suggested adding new and old election tuples to the error message. Did you decide not to do that? Or could you add an example of the suggested error message here?

Reply:
Nope, just missed that. Here's what the message looks like in the Node driver currently (implemented before we finalized this):

```
MongoStalePrimaryError: primary marked stale due to electionId/setVersion mismatch: server setVersion: 1, server electionId: 000000000000000000000001, topology setVersion: 1, topology electionId: 000000000000000000000002
```